### PR TITLE
Fix for launchFingerprintEnrollment

### PR DESCRIPTION
**Pull request by Flo354**
This PR fixes the issue #97

### CODE REVIEW

File: rxfingerprint/src/main/java/com/mtramin/rxfingerprint/RxFingerprint.java [hunk 1]
```diff
@@ -413,7 +415,7 @@ public boolean hasEnrolledFingerprints() {
     }
 
     public void launchFingerprintEnrollment() {
-		context.startActivity(new Intent("ACTION_FINGERPRINT_ENROLL"));
+		context.startActivity(new Intent(ACTION_FINGERPRINT_ENROLL));
```
Comment (Mauin) [inline rxfingerprint/src/main/java/com/mtramin/rxfingerprint/RxFingerprint.java]: As the constant is only available in the Android P dev preview, should we use the string constant directly instead? `"android.settings.FINGERPRINT_ENROLL"`

**Comment by Flo354** [comment]
I hesitated with this one. But then I saw that somewhere in the code you
use the class FingerprintDialog, which is also only available in Android P.

I think we can use directly the constant since this method should be called
only for P+.

But if you want I can use the static string and set a @todo to remember
that we have to modify this when Android P will be available in its final
release.

Le sam. 7 avr. 2018 à 9:25 AM, Marvin Ramin <notifications@github.com> a
écrit :

**Comment by Mauin** [comment]
You're right. I probably missed some more Android P-only classes that might be exposed. In any case the `launchFingerprintEnrollment` method should have a `@RequiresApi(P)` annotation to warn users to only call this on Android P as it won't be available before. Would be great if you can add it as well as part of this PR 👍

**Comment by Flo354** [comment]
I will do that.

It's another topic but isn't it possible to get a return from launchFingerprintEnrollment? So we can know when the user has added a fingerprint.

**Review by Mauin — APPROVED** [review]
Awesome, thanks!

**Pull request merged**